.github/workflows/benchmark.yml

```diff
@@ -24,12 +24,23 @@ jobs:
             ${{ runner.os }}-benchmark-${{ github.ref }}-${{ github.sha }}
             ${{ runner.os }}-benchmark-${{ github.ref }}
             ${{ runner.os }}-benchmark
-      - name: Compare benchmarks
-        uses: pksunkara/github-action-benchmark@v1
+      - name: Compare benchmarks for Pull Requests
+        uses: rhysd/github-action-benchmark@v1
+        if: github.event_name == 'pull_request'
         with:
           tool: cargo
           output-file-path: output.txt
           external-data-json-path: ./cache/benchmark-data.json
           github-token: ${{ github.token }}
           comment-always: true
-          save-on-pr: false
+          save-data-file: false
+      - name: Compare benchmarks for master branch
+        uses: rhysd/github-action-benchmark@v1
+        if: github.event_name == 'push'
+        with:
+          tool: cargo
+          output-file-path: output.txt
+          external-data-json-path: ./cache/benchmark-data.json
+          github-token: ${{ github.token }}
+          alert-threshold: 125%
```

Comment:
I think `alert-threshold: 125%` is way above the level. Probably should be 5%

Reply:
Github seems to use separate machines and thus us getting around 115% or 85% sometimes on master. This is to alleviate that.

Reply:
Then why do we use GActions in the first place if it's unreliable? 15% error is a lot. 

In the light of it, I feel like we should stop relying on GA and run the benches locally. New features don't land every day after all 🤷‍♂.

Reply:
Yup, we are not really taking these seriously. It's more like a check if some PR messes up performance badly.

Ideally we should have a stable runner somewhere in the cloud. Local benchmarks are not reliable since we run other things.

Reply:
It doesn't seem like this works with third-party PRs. Any idea why and how to fix?